contrib/vagrant/provision-util.sh

```diff
@@ -128,6 +128,9 @@ os::provision::set-os-env() {
   fi
   os::provision::set-bash-env "${origin_root}" "${config_root}" \
 "/root/${file_target}"
+
+  # Make symlinks to the bash completions for the openshift commands
+  ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
```

Comment:
It's called on line 121 of the Vagrantfile when
/vagrant/hack/dind-cluster.sh config-host is called on the host machine.

-ben

On Mon, Dec 14, 2015 at 3:22 PM, Maru Newby <notifications@github.com>
wrote:

> In contrib/vagrant/provision-util.sh
> <https://github.com/openshift/origin/pull/6256#discussion_r47552248>:
>
> > @@ -128,6 +128,9 @@ os::provision::set-os-env() {
> >    fi
> >    os::provision::set-bash-env "${origin_root}" "${config_root}" \
> >  "/root/${file_target}"
> > +
> > +  # Make symlinks to the bash completions for the openshift commands
> > +  ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
>
> set-os-env() is never called on the dind host, so as written this PR will
> only enable bash completion on vagrant nodes and dind nodes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6256/files#r47552248>.
>